src/js/stores/tool.js

```diff
@@ -202,15 +203,21 @@ define(function (require, exports, module) {
 
         /**
          * @private
-         * @param {{tool: Tool, keyboardPolicyListID: number, pointerPolicyListID: number}} payload
+         * @param {{tool: Tool, keyboardPolicyListID: number=, pointerPolicyListID: number=}} payload
          */
         _handleSelectTool: function (payload) {
             var tool = payload.tool;
 
             this._previousTool = this._currentTool;
             this._currentTool = tool;
-            this._currentKeyboardPolicyID = payload.keyboardPolicyListID;
-            this._currentPointerPolicyID = payload.pointerPolicyListID;
+
+            if (payload.keyboardPolicyListID) {
```

Comment:
during the handling of the "start" event, does it make sense to clear the current*policy vars?  Otherwise it seems like there is a period of inconsistency?  Actually, as I type this, I realize that this inconsistency properly reflects the actual (hopefully brief) inconsistent state that we're in.  So nevermind.